guides/common/modules/proc_enrolling-server-with-the-ad-server.adoc

```diff
@@ -18,7 +18,7 @@ Installing GSS-proxy and {nfs-client-package}:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {project-package-install} sssd adcli realmd ipa-python-compat krb5-workstation samba-common-tools
+# {project-package-install} adcli ipa-python-compat krb5-workstation oddjob oddjob-mkhomedir realmd samba-common-tools sssd
```

Comment:
Confirmed in https://issues.redhat.com/browse/SAT-26355 and in [RHEL AD integration docs](https://issues.redhat.com/browse/SAT-26355).

Reply:
Also requested in SAT-22855.